test/test_assembly.py

```diff
@@ -15,10 +15,13 @@
 from __future__ import print_function, division, absolute_import
 
 from isovar.variant_reads import reads_supporting_variant
-from isovar.variant_sequences import initial_variant_sequences_from_reads
+from isovar.variant_sequences import (
+    initial_variant_sequences_from_reads,
+    VariantSequence
```

Comment:
I combined two different import lines into one. (see the deleted line below). 

I can try to keep these kinds of changes out of PRs but it's also the kind of small cleanup that would never happen otherwise (since it's hard to remember to do it in some other PR).